server/explore.js

```diff
@@ -1,79 +0,0 @@
-/* explore.js
```

Comment:
good catch! I think i deleted the file in my branch and it didn't re add it on merge.